modules/priceFloors.js

```diff
@@ -113,6 +113,14 @@ export function getFirstMatchingFloor(floorData, bidObject, responseObject = {})
     matchingData: allPossibleMatches[0], // the first possible match is an "exact" so contains all data relevant for anlaytics adapters
     matchingRule
   };
+
+  // check that floorMin is greater than 0 and set values accordingly
+  if (utils.deepAccess(_floorsConfig, 'floorMin')) {
+    matchingData.floorMin = utils.deepAccess(_floorsConfig, 'floorMin');
+    matchingData.floorRuleValue = matchingData.matchingFloor;
+    if (matchingData.floorMin > matchingData.floorRuleValue) matchingData.matchingFloor = matchingData.floorMin;
+  }
+
```

Comment:
So I think from now on we want to always set the `floorRuleValue` so that needs to be done outside of this if.

Also no need for deepAccess.

Use `floorData` not `_floorsConfig` as the min may change between auctions.

I think something like this would work:


Rewriting lines 111 and on:

```javascript
let matchingData = {
    matchingData: allPossibleMatches[0], // the first possible match is an "exact" so contains all data relevant for anlaytics adapters
    matchingRule,
    floorRuleValue: floorData.values[matchingRule] || floorData.default, // we know the Rule Value always so need it always
    floorMin: floorData.floorMin || 0 // if not defined default it to 0 (this is only for comparison, not signaled to adapters)
  };
  // now we can set the actual floor to be enforced by picking the max of the two
  matchingData.matchingFloor = Math.max(floorData.floorMin, matchingData.floorRuleValue);

```